django-stubs/contrib/auth/__init__.pyi

```diff
@@ -27,7 +32,7 @@ def login(
 ) -> None: ...
 def logout(request: HttpRequest) -> None: ...
 def get_user_model() -> Type[Model]: ...
-def get_user(request: HttpRequest) -> Union[AbstractBaseUser, AnonymousUser]: ...
+def get_user(request: _HasSession) -> Union[AbstractBaseUser, AnonymousUser]: ...
```

Comment:
Thanks! 

I am just wondering if `Union[HttpRequest, TestClient]` is better.

Reply:
no problem!

i had that thought too. i thought iot might be better to do a protocol in case anyone wanted to implement something that acted like a client/request, but thinking on it now, i don't think that would be that common. using a union would definitely be easier to read